### ISSUE TITLE
awk: syntax error installing terraform-ls on MacOS

### ISSUE THREAD

**Issue by bobdoah** [mattn/vim-lsp-settings#540]
MacOS comes with a version of awk which doesn't support the third, array, argument to the match function. This is a GNU extension.

The install fails with:
```
awk: syntax error at source line 1
 context is
        match($0, /"tag_name": >>>  "(.+)"/, <<<
awk: bailing out at source line 1
~
```

[The change to introduce arch compatability](https://github.com/mattn/vim-lsp-settings/commit/386d493ff17513d57b11fb5ccca4363bb7fa9b44) introduced this issue.

**Comment by mattn** [comment]
@halostatue can you handle this issue?

**Comment by bobdoah** [comment]
@mattn I've put together a PR which undoes the incompatible part of @halostatue's change.

**Comment by halostatue** [comment]
PR #541 works as expected. I’ll need to suggest the same thing for tfvm.

**Comment by mattn** [comment]
Closed by #541
